packages/inter-protocol/test/auction/test-scheduler.js

```diff
@@ -654,3 +654,101 @@ test('duration = freq', async t => {
   };
   t.deepEqual(schedule.nextAuctionSchedule, secondSchedule);
 });
+
+test('change Schedule', async t => {
+  const { zoe } = await setupZCFTest();
+  const installations = await setUpInstallations(zoe);
+  /** @type {TimerService & { advanceTo: (when: Timestamp) => void; }} */
+  const timer = buildManualTimer();
+  const timerBrand = await timer.getTimerBrand();
+
+  const fakeAuctioneer = makeFakeAuctioneer();
+  const { fakeInvitationPayment } = await getInvitation(zoe, installations);
+  const publisherKit = makeGovernancePublisherFromFakes();
+
+  const { makeRecorderKit, storageNode } = prepareMockRecorderKitMakers();
+  const recorderKit = makeRecorderKit(storageNode);
+  let defaultParams = makeDefaultParams(fakeInvitationPayment, timerBrand);
+  // start hourly, request 6 steps down every 10 minutes, so duration would be
+  // 1 hour. Instead, cut the auction short.
+  defaultParams = {
+    ...defaultParams,
+    priceLockPeriod: 20n,
+    startFreq: 360n,
+    auctionStartDelay: 5n,
+    clockStep: 60n,
+    startingRate: 100n,
+    lowestRate: 40n,
+    discountStep: 10n,
+  };
+  const params = makeAuctioneerParams(defaultParams);
+  const params2 = {};
+  for (const key of Object.keys(params)) {
+    const { value } = params[key];
+    params2[key] = value;
+  }
+
+  await timer.advanceTo(127n);
+
+  const paramManager = await makeAuctioneerParamManager(
+    publisherKit,
+    zoe,
+    // @ts-expect-error 3rd parameter of makeAuctioneerParamManager
```

Comment:
i count four places that option bag is used. please pull it out into a named type and use it here.

actually there's already a type `AuctionParams`, though it uses the param names (e.g. `PriceLockPeriod` ). Imo that would be an improvement over the case change this is doing. Looking further, I think `makeAuctioneerParamManager` type signature is wrong because the code is expecting `initial['PriceLockPeriod']` but the type signature says,
```js
 * @param {RelativeTime} initial.priceLockPeriod
```

so it'll be more accurate to change that to,
```js
/**
 * @param {import('@agoric/notifier').StoredPublisherKit<GovernanceSubscriptionState>} publisherKit
 * @param {ZoeService} zoe
 * @param {AuctionParams} initial
 */
export const makeAuctioneerParamManager = (publisherKit, zoe, initial) => {
```
then you can type this third argument with that and drop the error suppression

Reply:
done.